tokio-epoll-uring/src/system/slots.rs

```diff
@@ -69,6 +69,23 @@ struct SlotsInner {
     unused_indices: Vec<usize>,
     co_owner_live: [bool; co_owner::NUM_CO_OWNERS],
     state: SlotsInnerState,
+    testing: SlotsTesting,
```

Comment:
```suggestion
    #[cfg(test)]
    testing: SlotsTesting,
```

Feel free to disregard if I am missing some grander plan.

Reply:
Yeah, wanted to add that, forgot it in the rush.